tests/app/upload/test_rest.py

```diff
@@ -0,0 +1,180 @@
+from datetime import datetime, timedelta, date
+
+from freezegun import freeze_time
+
+from app.models import LETTER_TYPE, JOB_STATUS_FINISHED, JOB_STATUS_PENDING
+from tests.app.db import create_job, create_notification, create_template, create_ft_notification_status
+from tests.conftest import set_config
+
+
+def create_uploaded_letter(letter_template, service, status='created', created_at=None):
```

Comment:
These two helper methods are duplicated in the DAO tests. Would be good to remove the duplication